_conferences/come-to-code-2023.md

```diff
@@ -0,0 +1,13 @@
+---
+name: "Come To Code"
+website: https://www.cometocode.it/
+location: Pignola, Italy
```

Comment:
La location e' corretta? Dal sito sembra sia Campobasso?

Reply:
Si, è Pignola in provincia di Potenza 

Usando il formato della location non sapevo se aggiungere o meno la provincia

Reply:
magari mettiamo:
```suggestion
location: Pignola (Potenza), Italy
```